stock_move_line_serial_unique/tests/test_stock_move_line_serial_unique.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestStockQuantSerialUnique(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.stock_location = cls.env.ref("stock.stock_location_stock")
+        cls.vendor_location = cls.env.ref("stock.stock_location_suppliers")
+        cls.picking_type_in = cls.env.ref("stock.picking_type_in")
+        cls.picking_type_out = cls.env.ref("stock.picking_type_out")
+        cls.owner1 = cls.env["res.partner"].create({"name": "Test Company"})
+        cls.product1 = cls.env["product.product"].create(
+            {"name": "Product A", "type": "product", "tracking": "serial"}
+        )
+        Serial1 = cls.env["stock.lot"].create(
+            {
+                "company_id": cls.env.company.id,
+                "product_id": cls.product1.id,
+                "name": "Serial1",
```

Comment:
```suggestion
        serial1 = cls.env["stock.lot"].create(
            {
                "company_id": cls.env.company.id,
                "product_id": cls.product1.id,
                "name": "001",
```
Avoid applying the camel case in a random manner.